benchmark/utils/timer.hpp

```diff
@@ -106,4 +106,25 @@ std::shared_ptr<Timer> get_timer(std::shared_ptr<const gko::Executor> exec,
     return std::make_shared<CpuTimer>(exec);
 }
 
+
+#if GINKGO_BUILD_MPI
+
+
+/**
+ * Get the timer. If the executor does not support gpu timer, still return the
+ * cpu timer.
```

Comment:
We should probably mention that the timer needs to be synchronous across all MPI ranks and returns the maximum.